### DISCUSSION TITLE
how do I build the shell script?

### DISCUSSION THREAD

**Discussion by haizhou0410** [mdolab/MACH-Aero#128] [Q&A]
Hi, 
I was installing the third party packages for mach aero. when I install PETS, I see it seems that I need to build a script. may I know how to build this and what to do after I build this. Thank you very much!

**Comment by haizhou0410** [comment]
![Uploading 17001023595562264283973932387820.jpg…]()

**Comment by eirikurj** [comment]
Hi, please update the problem description as its incomplete.
